config/openGemini.singlenode.conf

```diff
@@ -13,6 +13,7 @@
   flight-address = "127.0.0.1:8087"
   flight-enabled = true
   flight-auth-enabled = false
+  max-line-size = 65536
```

Comment:
Don't need to define `max-line-size = 65536` for the default behavior. Just remove it please.